src/eventHandlers/invocationRequest.ts

```diff
@@ -119,9 +119,11 @@ export function invocationRequest(channel: WorkerChannel, requestId: string, msg
         if (result && typeof result.then === 'function') {
             result
                 .then((result) => {
+                    //eslint-disable-next-line deprecation/deprecation

```

Comment:
Any chance we can get https://github.com/Azure/azure-functions-nodejs-worker/pull/547 merged first? This code is completely changing

Reply:
Sure, I'm happy to wait on merging this PR until #547 gets merged. I tried to review that PR but honestly I don't super follow what's going on, so I'd rather defer until @alrod can take a look.